README.md

```diff
@@ -49,6 +49,10 @@ If `LIBPCAP_LIBDIR` environment variable is set when building the crate, it will
 
 The crate will automatically try to detect the installed `libpcap`/`wpcap` version by loading it during the build and calling `pcap_lib_version`. If for some reason this is not suitable, you can specify the desired library version by setting the environment variable `LIBPCAP_VER` to the desired version (e.g. `env LIBPCAP_VER=1.5.0`). The version number is used to determine which library calls to include in the compilation.
 
+## Minimum Supported Rust Version (MSRV)
+
+Currently 1.41.0, talk about policies about MSRV is on https://github.com/rust-pcap/pcap/issues/231
```

Comment:
Update link to https://github.com/rust-pcap/pcap/discussions/240. I changed it to discussion as an issue is something that should have a well-defined objective and be closeable. A discussion can just stay open as long as people have something to contribute.